meta-luneos/recipes-webos-ose/maliit-framework-webos/files/maliit-server@.service

```diff
@@ -15,15 +15,16 @@
 # SPDX-License-Identifier: Apache-2.0
 
 [Unit]
-Description=webos - "%n", instance %i
+Description=meta-webos - "%n", instance %i
 Requires=maliit-server.service
 After=maliit-server.service
 BindsTo=maliit-server.service
 
 [Service]
 Type=simple
 OOMScoreAdjust=0
-EnvironmentFile=-/etc/maliit/maliit-env.conf
+Environment=QT_QPA_PLATFORM=wayland-egl
```

Comment:
mmh it's not "webos-wayland"? TBC

Reply:
This one is completely weird.... I see this as global:

https://github.com/webosose/webos-initscripts/blob/9610984b392d5bfb52ad45382f98da81b1c074c7/common/etc/systemd/system.conf.d/webos-global.conf.in#L44 and https://github.com/webosose/qml-webos-framework/blob/a1e2815af00e1d0dc065f8fc8656f58cdfc1635e/tests/launcher/main.cpp#L38

Reply:
It's possible this doesn't use the wayland extensions that webos-wayland provides, but if it works with webos-wayland, i'd probably remove that override, just so there are fewer overridden variables than need to be.  it might be possible also that giving it webos-wayland might break it if it doesn't know how to deal with the extensions.  🤷‍♂️